pages.ru/common/asciidoctor.md

```diff
@@ -0,0 +1,20 @@
+# asciidoctor
+
+> Преобразователь AsciiDoc файлов в другие форматы для публикации.
+> Дополнительная информация: <https://docs.asciidoctor.org>.
+
+- Преобразовать данный `.adoc` файл в HTML (формат на выходе по умолчанию):
+
+`asciidoctor {{path/to/file.adoc}}`
```

Comment:
Could you also translate the file paths please? Though of course the extensions should remain the same.